.travis.yml

```diff
@@ -15,18 +15,19 @@ notifications:
 env:
   global: # default values
     - ROS_DISTRO=melodic
-    - ROS_REPO=ros
+    - DOCKER_IMAGE=melodic-ci
+    - MOVEIT_BRANCH=melodic-devel
```

Comment:
I don't think it make sense to specify the `MOVEIT_BRANCH` here. This is only used for UPSTREAM_WORKSPACE, which in turn is a user-specified variable. So, if you like, you can already specify an arbitrary branch.